### PR TITLE
New US small business playbook

### PR DESCRIPTION
**Pull request by davidcardoza**
Mirroring a similar PR as https://github.com/Expensify/App/pull/14953

Tracking GH: https://github.com/Expensify/Expensify/issues/262473

!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->

### Details

Add a new article in the `Playbooks` hub in the help site.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/259739

### Tests

1. Open the `Playbooks` hub page (http://127.0.0.1:4000/hubs/playbooks on dev and https://help.expensify.com/hubs/playbooks on prod).
2. Click on `Expensify Playbook for US-Based VC-Backed Startups` and verify the article is loaded correctly.

- [x] Verify that no errors appear in the JS console

### Offline tests

N/A

### QA Steps

Same as `Test` steps

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos

The expensify help site only works on web

<details>
<summary>Web</summary>

Screenshot 2023-02-14 at 13 39 02

Screenshot 2023-02-14 at 13 39 11

</details>

</details>

### CODE REVIEW

**Comment by mananjadhav** [comment]
@davidcardoza I don't think this needs a C+ review. Feel free to unassign or let me know if you need me to review.

File: SmallbusinessPlaybook [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+1. Go to *Settings > Account*, and select the *Payments* tab
+2. Select *Add Verified Bank Account*
+3. From here, we’ll ask you to use your online banking credentials to connect to your bank (Note that this must be the account owner or admin credentials)
+4. Alternatively, you can go the more manual route by selecting “Connect Manually”
+5. Once that’s done, we’ll collect all of the necessary information on your business, such as your legal business name and address
+6. We’ll then collect your personal information, and a photo ID to confirm your identity
+
+You only need to do this once: you are fully set up for not only reimbursing expense reports, but issuing Expensify Cards, collecting customer invoice payments online (if applicable), as well as paying supplier bills online.
+
+# Step 6: Invite employees
```
Comment (deetergp) [inline SmallbusinessPlaybook:93]:
```suggestion
## Step 6: Invite employees
```

File: SmallbusinessPlaybook [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+- [Unique Policy Link](https://community.expensify.com/discussion/4643/how-to-invite-people-to-your-policy-using-a-join-link) - Each policy has a unique policy invite link, which is located at the top of the People tab in your policy settings. Simply share that link with anyone you’d like to add to your policy. 
+- [Manually](https://community.expensify.com/discussion/4975/how-to-invite-users-to-your-policy-manually-or-in-bulk/p1?new=1) - Enter employee email addresses manually by clicking the green Invite button in the People tab of your policy
+- [Google SSO](https://community.expensify.com/discussion/4774/how-to-enable-google-apps-sso-with-your-expensify-group-policy) - Or, if you have a Google Workspace configured, you can synchronize your policy's people list to match your Google Workspace employee list. 
+
+In the next section, we’ll go through how to configure approval routing but it’s important to remember that you’ll always have these 3 options to utilize, specifically the unique policy link and manual invites as your team continues to grow. 
+
+# Step 7: Set a Report Approver
```
Comment (deetergp) [inline SmallbusinessPlaybook:102]:
```suggestion
## Step 7: Set a Report Approver
```

File: SmallbusinessPlaybook [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+1. Go to *Settings > Policies*, and select the *People* tab.
+2. From there, select [Submit & Approve](https://community.expensify.com/discussion/5643/deep-dive-submit-and-approve) - this will automatically add you as the approver, which ensures that any expenses that fall outside of the rules you set for your policy are brought to your attention. If you want to change this to someone else (eg. your accountant) simply invite them to the policy and set them as the approver
+3. Looking for more advanced approval workflows? Reach out to your Setup Specialist to understand the features available when upgrading to Expensify’s Control Plan. 
+
+Thanks to our [Concierge Receipt audit technology](https://community.expensify.com/discussion/5738/deep-dive-how-does-concierge-receipt-audit-work), once you set up an approval workflow, most expenses will be audited automatically and won’t require manual review, however you can elect to manually approve expense reports too. Your time is valuable, so you should focus it on reviewing only the expenses that fall outside of your policy’s rules.  
+
+# Step 8: Set up your corporate card and assign cards to employees
```
Comment (deetergp) [inline SmallbusinessPlaybook:111]:
```suggestion
## Step 8: Set up your corporate card and assign cards to employees
```

File: SmallbusinessPlaybook [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+Expensify is optimized to work with corporate cards from all banks – or even better, use our own perfectly integrated Expensify Card.If you have an existing corporate card Expensify supports direct card feeds from most financial institutions. Setting up a corporate card feed will pull in the transactions from the connected cards on a daily basis. To set this up, do the following:
```
Comment (deetergp) [inline SmallbusinessPlaybook:112]:
```suggestion
Expensify is optimized to work with corporate cards from all banks – or even better, use our own perfectly integrated Expensify Card. If you have an existing corporate card Expensify supports direct card feeds from most financial institutions. Setting up a corporate card feed will pull in the transactions from the connected cards on a daily basis. To set this up, do the following: 
```

File: SmallbusinessPlaybook [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+1. Go to *Settings > Domains > Company Cards >* Select your bank 
+2. If you don’t see your financial institution in the list of banks we support, you can review an alternative solution in the Feature Deep Dives section below
+3. Next, enter your bank account login credentials.
+4. To successfully connect to your bank, we’ll need the *master admin (primary) account* login credentials. 
+5. Next, assign the corporate cards to your employees by selecting the employee’s email address and the corresponding card number from the two drop-down menus under the *Assign a Card* section
+6. Set a transaction start date (this is really important to avoid pulling in multiple outdated historical expenses that you don’t want employees to submit)
+
+As mentioned above, we’ll be able to pull in transactions as they post (daily) and handle receipt matching for you and your employees. One benefit of the Expensify Card for your company, is being able to  see transactions at the point of purchasewhich provides you with real time compliance. We even send users push notifications to SmartScan their receipt when it’s required and generate IRS compliant e-receipts as a backup wherever applicable.
```
Comment (deetergp) [inline SmallbusinessPlaybook:121]:
```suggestion
As mentioned above, we’ll be able to pull in transactions as they post (daily) and handle receipt matching for you and your employees. One benefit of the Expensify Card for your company, is being able to see transactions at the point of purchase which provides you with real time compliance. We even send users push notifications to SmartScan their receipt when it’s required and generate IRS compliant e-receipts as a backup wherever applicable. 
```

File: SmallbusinessPlaybook [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+2. Assign the cards to your employees
+- Set *SmartLimits*:
+- *Employees* - We recommend a low limit for most employees, roughly double the size of the maximum daily spend – such as $1000.  
+- *Execs* - We recommend a higher limit for executives, roughly 10x the limit of a non-executive employee (eg, $10,000).
+
+Once the Expensify Cards have been assigned, each employee will be prompted to enter their mailing address so they can receive their physical card. In the meantime, a digital card will be ready to use immediately.
+
+If you have an accounting system we directly integrate with, check out how we take automation a step further with [Continuous Reconciliation](https://community.expensify.com/discussion/7335/faq-what-is-the-expensify-card-auto-reconciliation-process). We’ll create an Expensify Card clearing and liability account for you. Each time settlement occurs, we’ll take the total amount of your purchases and create a journal entry that credits the settlement account and debits the liability account - saving you hours of manual reconciliation work at the end of your statement period. 
+
+# Step 9: Set up Bill Pay and Invoicing
```
Comment (deetergp) [inline SmallbusinessPlaybook:150]:
```suggestion
## Step 9: Set up Bill Pay and Invoicing
```

File: SmallbusinessPlaybook [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+As a small business, managing bills and invoices can be a complex and time-consuming task. Whether you receive bills from vendors or need to invoice clients, it's important to have a solution that makes the process simple, efficient, and cost-effective.
+
+Here are some of the key benefits of using Expensify for bill payments and invoicing:
+- Flexible payment options: Expensify allows you to pay your bills via ACH, credit card, or check, so you can choose the option that works best for you (US businesses only)
```
Comment (deetergp) [inline SmallbusinessPlaybook:154]:
```suggestion
- Flexible payment options: Expensify allows you to pay your bills via ACH, credit card, or check, so you can choose the option that works best for you (US businesses only).
```

File: SmallbusinessPlaybook [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+- Free, No Fees: The bill pay and invoicing features come included with every policy and workspace, so you won't need to pay any additional fees.
+- Integration with your business bank account: With your business bank account verified, you can easily link your finances to receive payment from customers when invoices are paid. .
```
Comment (deetergp) [inline SmallbusinessPlaybook:156]:
```suggestion
- Integration with your business bank account: With your business bank account verified, you can easily link your finances to receive payment from customers when invoices are paid.
```

File: SmallbusinessPlaybook [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+3. At the top of the bill, you’ll notice a Pay button. Once you click that, you’ll see options including ACH, credit/debit card, along with mailing a physical check.
+
+Similarly, you can send bills directly from Expensify as well. 
+
+1. From the *Reports* tab, select the down arrow next to *New Report* and select *Bill*
+2. Next, enter the Supplier’s email address, the Merchant name, the total amount, and the date
+3. At this point, you can also upload an attachment to further validate the bill if necessary
+4. Click *Submit*, we’ll forward the newly created bill directly to your Supplier.
+
+Reports, invoices, and bills - are largely the same in theory, just with different rules. As such, creating a customer invoice is just like creating an expense report and even a Bill.
```
Comment (deetergp) [inline SmallbusinessPlaybook:172]:
```suggestion
Reports, invoices, and bills are largely the same, in theory, just with different rules. As such, creating a customer invoice is just like creating an expense report and even a bill.
```

File: SmallbusinessPlaybook [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+
+1. From the *Reports* tab, select the down arrow next to *New Report* and select *Invoice*.
+2. Add all of the expenses/transactions tied to the Invoice
+3. Enter the recipient’s email address, a memo if needed, and a due date for when it needs to get paid, and click *Send*
+
+You’ll notice it’s a slightly different flow from creating a Bill. Here, you are adding the transactions tied to the Invoice, and establishing a due date for when it needs to get paid. If you need to apply any markups, you can do so from your policy settings under the Invoices tab. Your customers can pay their invoice in Expensify via ACH, or Check, or Credit Card. 
+
+# Step 10: Add a Payment card
```
Comment (deetergp) [inline SmallbusinessPlaybook:180]:
```suggestion
## Step 10: Add a Payment card
```

File: SmallbusinessPlaybook [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+Now that we’ve gone through all of the steps for setting up your account, let’s make it official so there are no interruptions in service as your employees begin using Expensify. We handle payments for our service via a paymentcard, and to add one:
+
+1. Go to *Account > Settings > Payments*
+2. Select *Add Payment Card*
+3. Enter your name, card number, postal code, expiration and CVV
+4. Click *Accept Terms*
+
+# You’re all set!
```
Comment (deetergp) [inline SmallbusinessPlaybook:188]:
```suggestion
## You’re all set!
```

File: SmallbusinessPlaybook [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+Congrats, you are all set up! If you need any assistance with anything mentioned above or would like to understand other features available in Expensify, reach out to either your Setup Specialist r directly in new.expensify.com. Don’t have one yet? Create a collect Policy, and we’ll automatically assign a dedicated Setup Specialist to you.
```
Comment (deetergp) [inline SmallbusinessPlaybook:189]:
```suggestion
Congrats, you are all set up! If you need any assistance with anything mentioned above or would like to understand other features available in Expensify, reach out to your Setup Specialist directly in new.expensify.com. Don’t have one yet? Create a collect Policy, and we’ll automatically assign a dedicated Setup Specialist to you. 
```

**Review by deetergp — CHANGES_REQUESTED** [review]
Overall looks good, just caught a few typos during my read-through.

**Comment by deetergp** [comment]
@parasharrajat @mananjadhav C+ reviews are not necessary for this, thanks for pointing it out 👍

**Comment by davidcardoza** [comment]
changes accepted

**Comment by davidcardoza** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by deetergp — APPROVED** [review]

**Comment by deetergp** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Comment by davidcardoza** [comment]
@deetergp should I be seeing this in help.Expensify.com now that it's in prod?

**Pull request merged**